src/components/conformance/utils.ts

```diff
@@ -15,17 +16,106 @@ export function createState(
   version: VersionItem,
   testPath?: string[],
   ecmaScriptVersion?: string,
+  sortOption?: string,
   selectedTest?: string,
 ): ConformanceState {
   testPath = testPath ? testPath : [version.tagName];
+  sortOption = sortOption ? sortOption : availableSortingOptions[0].id;
   return {
     version,
     testPath,
     ecmaScriptVersion,
+    sortOption,
     selectedTest,
   };
 }
 
+export const availableSortingOptions: SortOption[] = [
+  {
+    id: "alpha",
+    name: "Alphabetical",
+    callback: (a, b) => a.name.localeCompare(b.name),
+  },
+  {
+    id: "reverse",
+    name: "Reverse Alpha",
+    callback: (a, b) => -a.name.localeCompare(b.name),
+  },
+  {
+    id: "most-passed",
+    name: "Most Passed",
+    callback: (a, b) => b.stats.passed - a.stats.passed,
+  },
+  {
+    id: "least-passed",
+    name: "Least Passed",
+    callback: (a, b) => a.stats.passed - b.stats.passed,
+  },
+  {
+    id: "most-pass-percentage",
+    name: "Most Passed (%)",
+    callback: (a, b) =>
+      b.stats.total -
+      a.stats.total +
+      b.stats.passed / b.stats.total -
+      a.stats.passed / a.stats.total,
+  },
+  {
+    id: "least-pass-percentage",
+    name: "Least Passed (%)",
+    callback: (a, b) =>
+      b.stats.total -
+      a.stats.total +
+      a.stats.passed / a.stats.total -
+      b.stats.passed / b.stats.total,
+  },
+  {
+    id: "most-ignored",
+    name: "Most Ignored",
+    callback: (a, b) => b.stats.ignored - a.stats.ignored,
+  },
+  {
+    id: "least-ignored",
+    name: "Least Ignored",
+    callback: (a, b) => a.stats.passed - b.stats.passed,
```

Comment:
This is where that bug comes from for `Least Ignored` (I forgot to switch over the property field to ignore)

```suggestion
    id: "least-ignored",
    name: "Least Ignored",
    callback: (a, b) => a.stats.ignored - b.stats.ignored,
```